CONTRIBUTING.md

```diff
@@ -24,6 +24,7 @@
     1. [Writing Acceptance tests](#writing-acceptance-tests)
 1. [Using the Acceptance Test Framework to Debug](#using-acceptance-test-framework-to-debug)
 1. [Helm Reference Docs](#helm-reference-docs)
+1. [Managing External CRD Dependencies](#managing-external-crd-dependencies)
```

Comment:
quick drive by question 🚗: Currently, these CRDs will always get installed with a helm install. We should add an `if` condition around them as we do with the other CRDs to ensure they only get installed on a cluster if the gateway is enabled. Was this change intentional?

Reply:
I guess it is ok for this PR since we arent merging it into `main` but it is definitely something we should consider before that happens. This looks really clever otherwise. Nicely done.

Reply:
This is using the Helm CRDs directory, not our normal CRDs method. That directory doesn't support templating.

Reply:
Aaha! so this needs an explicit install! thanks for the clarification!

Reply:
Good call out though! It's a pain and I tried to get around it, but there really wasn't a way to do it :shrug: